pytorch_lightning/utilities/imports.py

```diff
@@ -79,6 +79,7 @@ def _compare_version(package: str, op, version) -> bool:
 _HOROVOD_AVAILABLE = _module_available("horovod.torch")
 _HYDRA_AVAILABLE = _module_available("hydra")
 _HYDRA_EXPERIMENTAL_AVAILABLE = _module_available("hydra.experimental")
+_JSONARGPARSE_AVAILABLE = _module_available("jsonargparse") and _compare_version("jsonargparse", operator.ge, "3.17.0")
```

Comment:
Why is the extra condition necessary? We don't do it for our other packages

```suggestion
_JSONARGPARSE_AVAILABLE = _module_available("jsonargparse")
```

Reply:
they changed API from 3.16 to 3.17 so if you had installed 3.16, all would crash for you know... but probably not needed as it is user bad that did not update env with requirements...